instagram-clone/src/App.test.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import App from './App';
+
+it('renders without crashing', () => {
```

Comment:
This is the default test file, I was confused for a second and thought you were writing tests with jest. Test passes 😀